js/apps/keycloak-server/scripts/start-server.js

```diff
@@ -9,30 +9,41 @@ import path from "node:path";
 import { pipeline } from "node:stream/promises";
 import { fileURLToPath } from "node:url";
 import { extract } from "tar-fs";
+import { parseArgs } from "node:util";
 
 const DIR_NAME = path.dirname(fileURLToPath(import.meta.url));
 const SERVER_DIR = path.resolve(DIR_NAME, "../server");
+const LOCAL_QUARKUS = path.resolve(DIR_NAME, "../../../../quarkus/dist/target");
+const LOCAL_DIST_NAME = "keycloak-999.0.0-SNAPSHOT.tar.gz";
 const SCRIPT_EXTENSION = process.platform === "win32" ? ".bat" : ".sh";
 const ADMIN_USERNAME = "admin";
 const ADMIN_PASSWORD = "admin";
 const AUTH_DELAY = 5000;
 const AUTH_RETRY_LIMIT = 3;
 
+const options = {
+  local: {
+    type: "boolean",
+  },
+};
+
 await startServer();
 
 async function startServer() {
-  await downloadServer();
+  let { values: scriptArgs, args: keycloakArgs } = handleArgs(
```

Comment:
Rather than using destructuring to alias the return values, perhaps you could return them with the same name from `handleArgs()`?

```suggestion
  const { scriptArgs, serverArgs } = handleArgs(process.argv.slice(2));
```